
The usual sloppy code gcc >= 3.4.x discovers.

  - Rene Rebe <rene@exactcode.de>

--- gnome-mag-0.11.9/magnifier/magnifier.c	2004-11-02 15:21:40.000000000 +0000
+++ gnome-mag-0.11.9-fixed/magnifier/magnifier.c	2004-12-12 05:20:47.000000000 +0000
@@ -1109,6 +1109,9 @@
 }
 
 static void
+magnifier_init_window (Magnifier *magnifier, GdkScreen *screen);
+
+static void
 magnifier_set_property (BonoboPropertyBag *bag,
 			BonoboArg *arg,
 			guint arg_id,
